src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioWorkspaceStatusServiceFactory.cs

```diff
@@ -141,6 +159,17 @@ private async Task EnsureInitializationAsync(CancellationToken cancellationToken
 
                     _initialized = true;
 
+                    _loadHubClientPackage = _threadingContext.JoinableTaskFactory.RunAsync(async () =>
+                    {
+                        await _threadingContext.JoinableTaskFactory.SwitchToMainThreadAsync(CancellationToken.None);
```

Comment:
possibly worth a comment stating why this is CT.None as opposed to the available CT passed in.  It feels like it could easily/accidentally use the wrong token and a commnt might help.  Up to you, and doesn't have to block merging if you'd like to move forward immediately on this.

Reply:
Switched to `_threadingContext.DisposalToken` and added a comment.